.github/workflows/openmdao_book_workflow.yml

```diff
@@ -20,53 +20,107 @@ jobs:
-          pip install nbconvert
-          pip install testflo
+          echo "=============================================================";
+          echo "Run #${GITHUB_RUN_NUMBER}";
+          echo "Run ID: ${GITHUB_RUN_ID}";
+          echo "Testing: ${GITHUB_REPOSITORY}";
+          echo "Triggered by: ${GITHUB_EVENT_NAME}";
+          echo "Initiated by: ${GITHUB_ACTOR}";
+          echo "=============================================================";
 
-      - name: 'Run testflo'
-        run: |
-          testflo .
+          echo "=============================================================";
+          echo "Create conda environment";
+          echo "=============================================================";
+          source $CONDA/etc/profile.d/conda.sh;
+          echo $CONDA/bin >> $GITHUB_PATH;
+          conda create -n OpenMDAO python=3.8 numpy=1.18 scipy=1.4 -q -y;
+          conda activate OpenMDAO;
 
-  # This job is called build_docs
-  build_docs:
-    # Run on Ubuntu
-    runs-on: ubuntu-latest
+          echo "=============================================================";
+          echo "Install PETSc";
+          echo "=============================================================";
+          conda install -c anaconda mpi4py -q -y;
+          conda install -c conda-forge petsc=3.12 petsc4py -q -y;
 
-    # Only run if test_docs was successful
-    needs: [ test_docs ]
+          echo "=============================================================";
+          echo "Install pyoptsparse";
+          echo "=============================================================";
 
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
-        with:
-          python-version: 3.8
+          git clone -q https://github.com/OpenMDAO/build_pyoptsparse;
 
-      - name: 'Install dependencies'
-        run: |
-          pip install matplotlib
-          pip install jupyter-book
-          pip install nbconvert
-          pip install testflo
+          cd build_pyoptsparse;
+          chmod 755 ./build_pyoptsparse.sh;
+          PAROPT=-a;
+          cd ..;
 
-      - name: 'Build OpenMDAO Book'
-        run: |
-          python build_jupyter_book.py
+          export LD_LIBRARY_PATH=$HOME/ipopt/lib;
+
+          echo "=============================================================";
+          echo "Install OpenMDAO";
+          echo "=============================================================";
+          cd ..;
+          git clone -q https://github.com/OpenMDAO/OpenMDAO;
+          cd OpenMDAO;
+          pip install .[all];
+          cd ../OpenMDAO_Book;
+
+          echo "=============================================================";
+          echo "Install optional packages for testing/coverage";
+          echo "=============================================================";
+          pip install psutil objgraph git+https://github.com/mdolab/pyxdsm;
+          pyppeteer-install;
```

Comment:
not sure if all these are needed for the docs?
(e.g. pyppeteer is for N2 testing)

Reply:
Yeah, we probably won't need these. I'll remove them and make sure it builds.